### ISSUE TITLE
CSS hashing problem

### ISSUE THREAD

**Issue by albertborsos** [rmrevin/yii2-minify-view#12]
I have some issue with the latest version (1.7.6)

When I modify a css file and then I reload the page, I got the same hash for the css files, like before the modification. Thats why the minified css will not overwritten, and the style is not refresh. And only 1 minified `.css` and `.js` files exist in the minified folder.

When I checkout to 1.7.3 it works. And on 1.7.4 if i modify the
```php
public $schemas = ['//', 'http://', 'https://', 'ftp://'];
```
line to
```php
public $schemas = ['/', 'http://', 'https://', 'ftp://'];
```
it works too.

The further changes was too complex, but something went wrong. Don't you realized this issue yet? Can you look after it? Thanks!

**Comment by rmrevin** [comment]
This bug has been fixed in version 1.7.8.
Thanks.

**Comment by albertborsos** [comment]
Uhhh sorry, I missed that release :D thanks :)

Az iPhone-omról küldve

2015. ápr. 17. dátummal, 17:33 időpontban Revin Roman <notifications@github.com> írta:
